### DISCUSSION TITLE
I want help with this problem

### DISCUSSION THREAD

**Discussion by smaile63** [shamhi/HamsterKombatBot#2281] [Q&A]
<img width="451" alt="image001" src="https://github.com/user-attachments/assets/0015dd24-4907-441e-9021-d1ad94fa605f">
![image]
![Uploading image001.png…]()

**Comment by cmeept** [comment]
![IMG_20240829_170528.jpg](https://github.com/user-attachments/assets/04ef497b-82c1-4f51-8ad5-003ad78281fe)

You must to install this app .

**Comment by STI228** [comment]
I had this problem and solved it by installing python 3.11
